libraries/GCS_MAVLink/GCS.h

```diff
@@ -1400,3 +1403,20 @@ enum MAV_SEVERITY
 
 #endif // HAL_GCS_ENABLED
 
+#if AP_MAVLINK_MSG_HIGHRES_IMU_ENABLED
```

Comment:
```suggestion
```

These defines don't hurt anything if they're unused. No need to wrap it in a define